### PR TITLE
Add typescript support and change color of calendar cell

### PR DESCRIPTION
**Pull request by jordanjones243**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #118, #125

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

* Add support for typscript users
* Change selected calendar-cell `hover` background color

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,60 +80,60 @@ In cases where the project is not able to process JS assets, there are pre-proce
 
 ### Bundle example code
 
-```html
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css" />
+```html
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/design-tokens@latest/dist/tokens/CSSCustomProperties.css" />
 <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@aurodesignsystem/webcorestylesheets@latest/dist/bundled/essentials.css" />
-<script src="https://cdn.jsdelivr.net/npm/@aurodesignsystem/auro-datepicker@latest/dist/auro-datepicker__bundled.js" type="module"></script>
+<script src="https://cdn.jsdelivr.net/@aurodesignsystem/auro-datepicker@latest/dist/auro-datepicker__bundled.js" type="module"></script>
```
Comment (jason-capsule42) [inline README.md:86]: This needs to be rebuilt using the latest template from WC-Generator. Just rerun the build:docs command.

**Review by jason-capsule42 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -27,6 +27,7 @@
     "@material/mwc-menu": "^0.27.0",
     "chalk": "^5.2.0",
     "lit": "^2.7.4",
+    "typescript": "^5.1.3",
```
Comment (blackfalcon) [inline package.json:30]: Not a dependency to use. Please move this to devDepedendeincies.

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Review by blackfalcon — CHANGES_REQUESTED** [review]
When building locally after `npm run sweep` I am getting the following error. 

```
ERROR: ./demo/apiExamples.md file is missing
```

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**